crml/cennzx-spot/src/impls.rs

```diff
@@ -142,7 +141,11 @@ pub(crate) mod impl_tests {
 				(price * fee_rate_factor) / scale_factor // price adjusted with fee
 			};
 
-			assert_eq!(core_asset_price, 539);
+			// core asset price is now calculated the same way whether we are buying
```

Comment:
I imagine this comment wouldn't make much sense later when there is no reference to the previous method of calculation. It's a good comment for the PR, though. So I suggest you take out from here.